playwright/tests/subjectSearch.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { Given, Then, When } from "@cucumber/cucumber";
+import { expect } from "@playwright/test";
+import { CustomWorld } from "../support/setup";
+
+Given("I want to do a subject search", async function (this: CustomWorld) {
+  return await this.page.goto("https://digital-research-books-beta.nypl.org/");
```

Comment:
Did you create a ticket to replace these with a QA url instead of prod? If so, can you link it here?

Reply:
And if tests are flaky on QA because things are slow, that should be mentioned at standup as something that engineers need to look into

Reply:
I did not create a ticket, but I implemented the functionality based on the example code you sent me.  At one of our working sessions, I like to discuss how to tighten that up event further.

I mentioned to team how prod is very slow at times.  It was acknowledged as an elastic search issue and will be addressed in coming sprints.